openstack/victoria/ovs-kernel/nova.patch

```diff
@@ -1,34 +1,37 @@
 diff --git a/nova/network/os_vif_util.py b/nova/network/os_vif_util.py
-index 0bfd87bb1e..e7c3c57593 100644
+index 0bfd87bb1e..5640b6cb49 100644
 --- a/nova/network/os_vif_util.py
 +++ b/nova/network/os_vif_util.py
-@@ -25,6 +25,7 @@ from oslo_log import log as logging
- from nova import exception
+@@ -26,6 +26,7 @@ from nova import exception
  from nova.i18n import _
  from nova.network import model
-+from nova.pci import utils as pci_utils
  
++from vif_plug_ovs import linux_net
```

Comment:
better to call it rom vif_plug_ovs import linux_net as os_vif_ linux_net as there is nova linux_net